tests/serverpod_test_server/test_integration/database_operations/entity_relations/entities_with_list_relations/order_by_test.dart

```diff
@@ -61,13 +163,45 @@ void main() async {
           ),
           db.Order(
             column: Person.t.name,
+          )
+        ],
+      );
+
+      var personNames = persons.map((e) => e.name);
+      expect(personNames, [
+        'Alex',
+        'Annie',
+        'Isak',
+        'Lisa',
+        'Marc',
+        'Viktor',
+        'Wendy',
+        'Jane',
+        'John',
+        'Tom'
+      ]);
+    });
+
+    test(
+        'when ordering on multiple many relations then result order is as expected.',
+        () async {
+      var cities = await City.db.find(
+        session,
+        // Order cities by number of citizens and then the number of organizations
```

Comment:
And here, check all the tests here!

Reply:
These tests should be updated now. Let me know what you think.

Reply:
I think the description describes the scenario that fails. These comments are just to understand the implementation details of the test.